lib/rules/hidden-content.js

```diff
@@ -0,0 +1 @@
+return axe.commons.dom.isVisible(node);
```

Comment:
This should check if the element has `display:none`, `visibility:hidden` or `aria-hidden=true`.